### DISCUSSION TITLE
Suggestion for editing monitors!

### DISCUSSION THREAD

**Discussion by FlamedDogo99** [ScratchAddons/ScratchAddons#4700] [Ideas]
Scratch addons covers many parts of scratch, however, the only way to currently edit monitors is to download and edit the project json. My suggestion is adding a dialog for editing what reporter a monitor is assigned to, similar to the edit variable dialog.
I’ve included an example project of an edited monitor. Instead of a variable, it’s been assigned to the counter reporter.
https://scratch.mit.edu/projects/639929567

**Comment by Oren-Lindsey** [comment]
so you mean you can make monitors for any variable, even hidden ones? If so, [already suggested](https://github.com/ScratchAddons/ScratchAddons/issues/4230)

**Comment by FlamedDogo99** [comment]
Thank you, hadn’t seen that.
